chart/templates/deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         - --registry-url={{ .Values.registry.url }}
         - --registry-org={{ .Values.registry.org }}
         - --registry-auth={{ .Values.registry.authtoken }}
+        - --docker-from-env=true
```

Comment:
Good point. I think we can actually remove this flag from the deployment and just [flip the default flag value](https://github.com/Azure/draft/blob/master/cmd/draftd/start.go#L64) to `true`. Would you mind doing that instead? :)